src/topology/basic.lean

```diff
@@ -605,11 +605,11 @@ if it exists. -/
 noncomputable def lim [nonempty α] (f : filter β) (g : β → α) : α :=
 Lim (f.map g)
 
-lemma Lim_spec {ha : nonempty α} {f : filter α} (h : ∃a, f ≤ 𝓝 a) : f ≤ 𝓝 (Lim f) :=
+lemma Lim_spec {f : filter α} (h : ∃a, f ≤ 𝓝 a) : f ≤ 𝓝 (@Lim _ _ (nonempty_of_exists h) f) :=
```

Comment:
You could maybe add a docstring mentioning the "real" content of the lemma (without the `@`) and explain why you formulate it in this way. Same thing for next lemma, and then it will be good to go.